src/utils/analysis-utils.ts

```diff
@@ -145,6 +176,7 @@ export type ApiData = {
   zones_url: string;
   group_by?: string;
   geojson_out?: boolean;
+  wfs_params?: { time?: string; layer_name: string; url: string } | {};
```

Comment:
what is `| {}` for? A comment to explain would be awesome :)

Reply:
I modified the type definition here 07e7f18f39d24bd8b96f0d26f5edd817278bc615. What happens is that we have to make the types in snake case since are request parameters sent to the backend. Please check this part. IMO we would not need to make two types: one as snake_case and another as CamelCase for the same variables.

Reply:
I agree, and your implementation looks good! All good to have an eslint disable here, make sure however that all types are in PascalCase

Reply:
Commit included in 4e2d594edc8628cdafed743586c55e02d78a176f